bundle/openshift/manifests/opentelemetry-operator-prometheus-rules_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: operator-lifecycle-manager
```

Comment:
Why is this label being added? Is it required for some use-case, does it fixes any bug? It's not being used in our bundle https://github.com/open-telemetry/opentelemetry-operator/tree/main/bundle/openshift/manifests

Reply:
I just removed them. I thought we wanted to have them since some objects have  them but others not

Reply:
>  since some objects have them

What objects have them? I didn't find it in https://github.com/open-telemetry/opentelemetry-operator/tree/main/bundle/openshift/manifests

Reply:
I just recheck the thing. It seems they are added later or I don´t know exactly where I saw them. But I removed that part of the PR anyway.